airbyte-integrations/connectors/source-babelforce/source_babelforce/manifest.yaml

```diff
@@ -0,0 +1,121 @@
+version: 0.50.2
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - calls
+
+streams:
+  - type: DeclarativeStream
+    name: calls
+    primary_key:
+      - id
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://{{ config['region'] }}.babelforce.com/api/v2/
+        path: calls/reporting
```

Comment:
I think the correct path should be `calls/reporting/simple`. The one using today is returning a csv format https://api.babelforce.com/#e0e4ec76-c14a-407c-bd77-17bc9ad29ab0

Reply:
Fixed now @marcosmarxm